docs/release/release-tasks.md

```diff
@@ -90,11 +90,20 @@ The goal of this issue is to bump the versions on the main branch so that the up
 is used for e.g. local development and e2e tests. We also modify tests so that they are testing the previous release.
 
 This comes down to changing occurrences of the old version to the new version, e.g. `v1.3` to `v1.4`:
-1. Add new release to top-level `metadata.yaml` and `test/e2e/data/shared/v1beta1/metadata.yaml`.
+1. Create a new `v1.3` `metadata.yaml` (`test/e2e/data/shared/v1.3/metadata.yaml`) by copying
+   `test/e2e/data/shared/main/metadata.yaml`
+1. Add the new release to the main `metadata.yaml` (`test/e2e/data/shared/main/metadata.yaml`).
```

Comment:
What about grouping items related to E2E under a single bullet? Something like
   
```suggestion
1. Add the new release to the main `metadata.yaml` (`test/e2e/data/shared/main/metadata.yaml`).
2. Setup E2E tests for the new release
   3. Create a new `v1.3` `metadata.yaml` (`test/e2e/data/shared/v1.3/metadata.yaml`) by copying
   `test/e2e/data/shared/main/metadata.yaml`
   .... (everything till L106)
   5. Make sure all tests are green (also run `pull-cluster-api-e2e-full-main` and `pull-cluster-api-e2e-workload-upgrade-1-23-latest-main`).
3. Update `create-local-repository.py` and `tools/tilt-prepare/main.go` with the new release: `v1.3.99` => `v1.4.99`.
```

Reply:
Good idea, done